packages/vow/src/tools.js

```diff
@@ -8,7 +8,36 @@ import { makeWhen } from './when.js';
 /**
  * @import {Zone} from '@agoric/base-zone';
  * @import {Passable} from '@endo/pass-style';
- * @import {IsRetryableReason, AsPromiseFunction, EVow, Vow, ERef} from './types.js';
+ * @import {EUnwrap, IsRetryableReason, AsPromiseFunction, Vow, VowKit, EVow, PromiseVow, Watcher} from './types.js';
+ */
+
+/**
+ * @typedef VowTools
```

Comment:
The formatting here looks kinda dense, so I looked for a typedoc rendering at https://ta-await-when.agoric-sdk.pages.dev/modules/_agoric_vow but I don't see one. hm.

Reply:
I'll clean this up in a future PR that moves them into a `.ts`